src/js/pages/Teams.jsx

```diff
@@ -81,14 +82,12 @@ const Teams = ({ classes, match }) => {
         </div>
         {teamList.map((team, index) => (
           <OneTeamWrapper key={`team-${team.id}`}>
-            <TeamHeader
-              team={team}
-              showHeaderLabels={(index === 0) && showAllTeamMembers}
-              // showHeaderLabels={(index === 0) && showAllTeamMembers && (team.teamMemberList && team.teamMemberList.length > 0)}
-              showIcons
-            />
+            <TeamHeader team={team} showHeaderLabels={(index === 0) && showAllTeamMembers && (team.teamMemberList && team.teamMemberList.length > 0)} showIcons />
```

Comment:
The rest of the team has standardized on putting each prop being passed into a component on its own line when we have 3+ props. It makes it easier to see what is being passed in at a glance.

Reply:
No problem, again a reversion/cherry pick